apps/metrics/urls.py

```diff
@@ -1,12 +1,20 @@
 from django.conf.urls import url
 from django.contrib import admin
-from .views import BeneMetricsView, AppMetricsView, AppMetricsDetailView, TokenMetricsView
+from .views import (
+    BeneMetricsView,
+    AppMetricsView,
+    AppMetricsDetailView,
+    TokenMetricsView,
+    DevelopersView,
+)
 
+# QUESTION: What is this?
```

Comment:
Found some info about the need to call admin.autodiscover() in urls.py being obsoleted in versions 1.7+. It is run via the admin app's default config now. So thinking we can try removing from here and the three other url.py files.